src/main/java/io/mosip/mimoto/exception/InvalidWellknownResponseException.java

```diff
@@ -0,0 +1,21 @@
+package io.mosip.mimoto.exception;
+
+public class InvalidWellknownResponseException extends BaseCheckedException {
+
+    public InvalidWellknownResponseException() {
+        super(PlatformErrorMessages.INVALID_CREDENTIAL_ISSUER_WELLKNOWN_RESPONSE_EXCEPTION.getCode(),PlatformErrorMessages.INVALID_CREDENTIAL_ISSUER_WELLKNOWN_RESPONSE_EXCEPTION.getMessage());
+    }
+
+    public InvalidWellknownResponseException(String message) {
+        super(PlatformErrorMessages.INVALID_CREDENTIAL_ISSUER_WELLKNOWN_RESPONSE_EXCEPTION.getCode(),PlatformErrorMessages.INVALID_CREDENTIAL_ISSUER_WELLKNOWN_RESPONSE_EXCEPTION.getMessage()+"\n"+message);
```

Comment:
why are we appending the error message with "/n" when this is thrown to front end , it might have UI issues if directly displayed from UI. shall we remove it.

Reply:
I don't think we are throwing this to frontend. this is for mimoto logging purposes

Reply:
yeah i get that, usually error message is intended for frontend.

Reply:
yes. We added detailed logging to figure out the actual failure scenario , in mimoto logs. 👍